zerver/lib/send_email.py

```diff
@@ -44,6 +44,8 @@ def build_email(template_prefix, to_email, from_name=None, from_address=None,
         reply_to = reply_to_email
     else:
         reply_to = from_email
+    if parseaddr(reply_to)[1] == FromAddress.NOREPLY:
+        reply_to = FromAddress.NOREPLY
```

Comment:
Isn't this already implied by other code?  Or is this to catch cases we forgot to update to use NOREPLY properly?

I would prefer to modify our tests or otherwise audit that we're not passing the noreply email incorrectly than add this sort of cover-up thing; they tend to hide bugs.